### PR TITLE
Send coins and withdraw funds

### PR DESCRIPTION
**Pull request by martineehrlich**
This adds the sendCoins lnd action that moves money from the lnd wallet to the specified address.
This adds the withdrawFunds engine action that is specifically allows a user to withdraw funds from their lnd wallet.

### CODE REVIEW

**Comment by martineehrlich** [comment]
[withdraw to address [ccxt]](https://trello.com/c/9tYwifnd/660-withdraw-to-address-ccxt)

File: src/engine-actions/withdraw-funds.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const {
+  sendCoins
+} = require('../lnd-actions')
+
+/**
+ * Given a payment request, it pays the invoices and returns a refund invoice
```
Comment (dannypaz) [inline src/engine-actions/withdraw-funds.js]: update the description

File: src/engine-actions/withdraw-funds.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * @param {String} addr wallet address to send the coins to
+ * @param {Integer} amount amount of coin to send to wallet address
+ * @param {Object} options
```
Comment (dannypaz) [inline src/engine-actions/withdraw-funds.js]: can remove this

File: src/engine-actions/withdraw-funds.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * @return {String} txid transaction for the withdrawal
+ */
+
+async function withdrawFunds (addr, amount) {
+  try {
+    const { txid } = await sendCoins(addr, amount, { client: this.client })
+    this.logger.debug('Coins successfully sent', { txid })
```
Comment (dannypaz) [inline src/engine-actions/withdraw-funds.js]: [nit] update this to be something like `Funds withdrawn successfully` to match the method name

File: src/lnd-actions/send-coins.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * @function
+ * @see {@link http://api.lightning.community/#sendCoins}
+ * @param {String} addr
+ * @param {Integer} amount
+ * @return {Object} response
+ * @return {String} response.txid
+ */
+function sendCoins (addr, amount, { client }) {
+  console.log('SENDCOINS APRM', addr, amount)
```
Comment (dannypaz) [inline src/lnd-actions/send-coins.js]: remove dev log

File: src/lnd-actions/send-coins.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+const { deadline } = require('../grpc-utils')
+
+/**
+ * Check's an invoice status
```
Comment (dannypaz) [inline src/lnd-actions/send-coins.js]: update info

File: src/engine-actions/withdraw-funds.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+ * Given a payment request, it pays the invoices and returns a refund invoice
+ *
+ * @param {String} addr wallet address to send the coins to
+ * @param {Integer} amount amount of coin to send to wallet address
+ * @param {Object} options
+ * @return {String} txid transaction for the withdrawal
+ */
+
+async function withdrawFunds (addr, amount) {
+  try {
```
Comment (dannypaz) [inline src/engine-actions/withdraw-funds.js]: Remove the try catch because it is redundant

**Comment by dannypaz** [comment]
LGTM @martineehrlich

**Pull request merged**